lms/views/error.py

```diff
@@ -73,16 +70,6 @@ def error(request):
 
 
 def includeme(config):
-    debug = asbool(os.environ.get("DEBUG") or config.registry.settings.get("debug"))
-    if debug:
-        # Don't register the error pages in development environments.  Let
-        # pyramid_debugtoolbar show the traceback in the browser and terminal
-        # instead.
-        # If you want to test the error pages in your dev env you can set the
-        # environment variable DEBUG to false:
-        #     export DEBUG=false
-        return
-
```

Comment:
:+1: Random different behavior in dev makes testing a nightmare

Reply:
Yeah this has been a continual PITA